mix.exs

```diff
@@ -7,11 +7,10 @@ defmodule EctoJob.Mixfile do
   def project do
     [
       app: :ecto_job,
-      description: "A transactional job queue built with Ecto, PostgreSQL and GenStage.",
       version: @version,
       elixir: "~> 1.10",
       elixirc_paths: elixirc_paths(Mix.env()),
-      elixirc_options: [warnings_as_errors: true],
+      elixirc_options: [warnings_as_errors: false],
```

Comment:
Why?

Reply:
> Why?

Accidentally change added while generating the doc. Reverted.